roles/docker_storage_to_overlay2/tasks/main.yml

```diff
@@ -0,0 +1,137 @@
+---
+
+# As of 7.4, SELinux security labels are now supported
+# on the OverlayFS file system (see RHEL release notes).
+- name: Check if distribution supports overlayfs (RHEL/CentOS)
+  when: ansible_distribution in ["RedHat", "CentOS"]
+  assert:
+    that: ansible_distribution_version | version_compare("7.4", "ge")
+    msg: "The OverlayFS file system only works with SELinux in {{ ansible_distribution }} 7.4 or later."
+
+- name: Install or update required packages
+  package:
+    name: "{{ item.key }}"
+    state: latest
+  with_dict: "{{ package_requirements }}"
+
+- name: Capture storage setup configuration
+  slurp:
+    src: "{{ storage_setup_path }}"
```

Comment:
This should move to a variable in `defaults/main.yml`.   Add role_prefix to it.

Reply:
@kwoodson as in:

``role_storage_setup_path``

or

``docker_storage_to_overlay2_storage_setup_path``